lib/fleiss/backend/active_record/concern.rb

```diff
@@ -4,6 +4,8 @@ class ActiveRecord
       module Concern
         extend ActiveSupport::Concern
 
+        DEFAULT_LOCK_TTL = ENV['FLEISS_LOCK_TTL']&.to_i # seconds
```

Comment:
No dynamic / official config (per-worker or so).

Considering this a tweak (even hack).

Reply:
OK, so now I see it, I don't like this. Let's add a proper heartbeart instead. How about:

1. add a `lock_expires_at` column - not sure how to deal with the migration of this?
2. document a `def max_run_time` callback (like we do with `def ttl`)
3. set `lock_expires_at` when `job.max_run_time` is set - much like https://github.com/bsm/fleiss/blob/main/lib/fleiss/backend/active_record/concern.rb#L46
4. fix `def pending` scope

Reply:
I find suggested solution viable but I don't like it much:

- if you call it `max_run_time` - that sounds like you imply that job can be killed when this is reached (`Time.now > started_at + max_run_time`)
- if you call it proper `lock_ttl` (which better be done globally, per worker) - this is kinda closer, but also not worth much without a proper heartbeat in a separate thread (maybe one per-worker thread, not per job)

I tend to implement second option (one heartbeat thread per worker).

Btw, migrations are solved by `create_table(...) unless table_exists?(...)` and `add_column(...) unless column_exists?(...)`.